appengine/test/test_myria_up.py

```diff
@@ -101,8 +104,17 @@ def test_datasets_connects():
     response = mock_get('/datasets')
     assert_equals(response.status_code, 200)
     assert 'fake.fake:12345 [2/2]' in str(response)
+    assert 'Create time' in str(response)
+
+
+def test_datasets_fetch():
+    response = mock_get('/datasets')
+    assert_equals(response.status_code, 200)
+    assert 'fake.fake:12345 [2/2]' in str(response)
     # Ensure it includes the Twitter dataset, creation time, and download URL
-    assert 'Twitter' in str(response)
+    # This test is requires javascript checking
+    raise SkipTest()
```

Comment:
The dataset and query tables are loaded through javascript ajax calls so the http response received does not include the data the tests checks for.